docs/modules/fault-tolerance/pages/backups.adoc

```diff
@@ -7,6 +7,15 @@ The distribution happens on the partition level; the data and its backups are st
 memory partitions. See the xref:overview:data-partitioning.adoc[Data Partitioning] and xref:clusters:partition-group-configuration.adoc[Partition Grouping]
 for more information about the partitioning.
 
+[IMPORTANT]
+====
+Hazelcast does not support backup redistribution on a data structure level. Thus, changing the backup count of a nonempty data structure is **not** supported. For example, for a map with a nonzero backup count:
+
+1. increasing the backup countfootnote:change-backup-count[The backup count of a map can only be changed by modifying the xref:configuration:understanding-configuration.adoc#static-configuration[static configuration] and restarting the cluster.] does **not** create additional copies of existing map entries,
```

Comment:
the footnote says that backup count can only be changed. While I guess what is supposed to mean, it looks contradictory and puzzling.

```suggestion
1. increasing the backup countfootnote:change-backup-count[The backup count of an empty map can only be changed by modifying the xref:configuration:understanding-configuration.adoc#static-configuration[static configuration] and restarting the cluster.] does **not** create additional copies of existing map entries,
```

Reply:
f1b97d8d